This provides the option to set the indicator for a NFS profile to true.
It may be necessary for users on NFS to use bookmarks/logging.
Needs further investigation, will probably be difficult to send upstream.

--- comm-esr31/mozilla/browser/app/profile/firefox.js.orig	2015-06-04 17:42:52.206758105 -0700
+++ comm-esr31/mozilla/browser/app/profile/firefox.js	2015-06-04 17:42:52.212124280 -0700
@@ -1022,6 +1022,9 @@
 // allow META refresh by default
 pref("accessibility.blockautorefresh", false);
 
+// Whether profile is on NFS
+pref("storage.nfs_filesystem", true);
+
 // Whether history is enabled or not.
 pref("places.history.enabled", true);
 
